rxconfig.py

```diff
@@ -15,5 +15,4 @@
             "@tailwindcss/typography",
         ],
     },
-    redis_url="localhost:6379",
```

Comment:
Assuming this was left in there by accident.

Reply:
good catch